src/encoded/static/components/genome_browser.js

```diff
@@ -331,9 +350,10 @@ class GenomeBrowser extends React.Component {
             trackObj.name = `${file.accession} ${file.output_type} ${file.biological_replicates ? `rep ${file.biological_replicates.join(',')}` : ''}`;
             trackObj.type = 'annotation';
             trackObj.path = domain + file.href;
-            // bigBed bedRNAElements have two tracks and need extra height
-            // There is inconsistency in the capitalization of the file format in the data
-            if ((file.file_type.toLowerCase() === 'bigbed bedrnaelements') || (file.file_format.toLowerCase() === 'bigbed bedrnaelements')) {
+            // bigBed bedRNAElements, bigBed peptideMapping, bigBed bedExonScore, bed12, and bed9 have two tracks and need extra height
+            // Convert to lower case in case of inconsistency in the capitalization of the file format in the data
+            if (file.file_format_type &&
+                ((file.file_format_type.toLowerCase() === 'bedrnaelements') || (file.file_format_type.toLowerCase() === 'peptidemapping') || (file.file_format_type.toLowerCase() === 'bedexonscore') || (file.file_format_type.toLowerCase() === 'bed12') || (file.file_format_type.toLowerCase() === 'bed9'))) {
```

Comment:
If this test fails, it will have converted the same variable to lowercase five times. Should just do that once.

Reply:
I fixed this.